Utils/Functions/Net/GithubReleases/script.js

```diff
@@ -1,29 +1,41 @@
 const Downloader = include("utils.functions.net.download");
-const {createTempDir, remove, cat} = include("utils.functions.filesystem.files");
+const Resource = include("utils.functions.net.resource");
+const { createTempDir, remove, cat } = include("utils.functions.filesystem.files");
 
 /**
- * Get an array of tag corresponding to the releases in a git repository
+ * Fetches an array of tag corresponding to the releases in a git repository
  *
  * @param {string} repositoryOwner owner of the repository
  * @param {string} repositoryName name of the repository
  * @param {wizard} wizard the wizard
- * @returns {string[]} list of version tag in the repository
+ * @returns {object[]} list of version objects in the repository
  */
-module.getGithubReleases = function getGithubReleases(repositoryOwner, repositoryName, wizard) {
+module.fetchGithubReleases = function fetchGithubReleases(repositoryOwner, repositoryName, wizard) {
     const tmpDir = createTempDir();
 
     const releasesFile = new Downloader()
         .wizard(wizard)
-        .url(
-            `https://api.github.com/repos/${repositoryOwner}/${repositoryName}/releases`
-        )
+        .url(`https://api.github.com/repos/${repositoryOwner}/${repositoryName}/releases`)
         .message(tr("Fetching versions list..."))
         .to(tmpDir + "/releases.json")
         .get();
 
-    const versions = JSON.parse(cat(releasesFile)).map(version => version.tag_name);
+    const versions = JSON.parse(cat(releasesFile));
 
     remove(tmpDir);
 
     return versions;
 };
+
+module.extractGithubReleaseStrings = function extractGithubReleaseStrings(versions) {
+    return versions.map(version => version.name);
+}
```

Comment:
Do we need to method?

Reply:
Is it used? I have no clue.

Reply:
It is currently but it is just a one-liner. It does not really add anything. In some cases it may also be a bad choice because `version.name` can contain the "wrong" release version string.